woocommerce-pdf-invoices-packingslips.php

```diff
@@ -273,11 +272,17 @@ public function need_woocommerce() {
 	/**
 	 * PHP version requirement notice
 	 */
-	
 	public function required_php_version() {
-		$error = __( 'WooCommerce PDF Invoices & Packing Slips requires PHP 5.3 or higher (5.6 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' );
-		$how_to_update = __( 'How to update your PHP version', 'woocommerce-pdf-invoices-packing-slips' );
-		$message = sprintf('<div class="error"><p>%s</p><p><a href="%s">%s</a></p></div>', $error, 'http://docs.wpovernight.com/general/how-to-update-your-php-version/', $how_to_update);
+		$message = sprintf(
+			'<div class="error"><p>%s</p><p>%s <a href="%s">%s</a> %s <a href="%s">%s</a>.</p></div>',
+			__( 'WooCommerce PDF Invoices & Packing Slips requires PHP 7.1 or higher (7.1 or higher recommended).', 'woocommerce-pdf-invoices-packing-slips' ),
+			__( 'You should', 'woocommerce-pdf-invoices-packing-slips' ),
+			'http://docs.wpovernight.com/general/how-to-update-your-php-version/',
+			__( 'update your PHP version', 'woocommerce-pdf-invoices-packing-slips' ),
+			__( 'or', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I think you should take a different approach here because translating parts of sentences may not work that well in all languages. You could also try:
```
You should %supdate your PHP version%s or %sdownload and install this add-on%s to enable PHP5.6 support
```
where you insert the `<a>` tags + url at the %s
or separated:
```
We strongly recommend to %supdate your PHP version%s.
If you cannot upgrade your PHP version you can download %sthis addon%s to enable backwards compatibility with PHP5.6
```